GitCommands/Git/Executable.cs

```diff
@@ -98,7 +128,15 @@ public ProcessWrapper(string fileName, string arguments, string workDir, bool cr
 
                 _process.Exited += OnProcessExit;
 
-                _process.Start();
+                try
+                {
+                    _process.Start();
+                }
+                catch (Exception ex)
+                {
+                    _logOperation.LogProcessEnd(null);
```

Comment:
What do you think about logging the exception here, so it can be viewed in the command log?
E.g.
```suggestion
                    _logOperation.LogProcessEnd(null, ex);
```

Reply:
> What do you think about logging the exception here, so it can be viewed in the command log?

👍 I will extend the classes `ProcessOperation` and CommandLogEntry.